export/export_pipeline/compute_collision_representation.py

```diff
@@ -0,0 +1,427 @@
+"""
+This script decomposes all static meshes into sets of convex regions.
+Decomposition method and parameters are based on the parameters chosen in UE, which are captured in 
+gltf_scene/actors_information.json.
+"""
+import argparse
+from collections import deque, defaultdict
+import copy
+import json
+import multiprocessing as mp
+if mp.current_process().name == 'MainProcess':
+  import bpy
+import coacd  # needs to be imported after bpy, otherwise terminate called after throwing an instance of 'std::bad_cast'
+import numpy as np
+import os
+import shutil
+import subprocess
+import trimesh
+import utils
+import yaml
+
+
+osp = os.path
+
+
+def decimation_lookup(poly_count):
+    # Get the adaptive decimation ratio
+    if poly_count < 10000:
+        return 1.0
+    elif poly_count < 50000:
+        return 0.8
+    elif poly_count < 100000:
+        return 0.6
+    elif poly_count < 500000:
+        return 0.5
+    elif poly_count < 1000000:
+        return 0.2
+    elif poly_count < 5000000:
+        return 0.1
+    elif poly_count < 10000000:
+        return 0.01
+    elif poly_count < 20000000:
+        return 0.009
+    elif poly_count < 30000000:
+        return 0.008
+    elif poly_count < 40000000:
+        return 0.009
+    elif poly_count < 50000000:
+        return 0.005
+    else:
+        return 0.001
+
+
+def get_relative_pose(obj):
+    """
+    gets pose relative to parent
+    """
+    obj.select_set(True)
+    xyz = copy.deepcopy(obj.location)
+    xyz = [xyz.x, xyz.y, xyz.z]
+    quat = copy.deepcopy(obj.rotation_quaternion)
+    quat = [quat.w, quat.x, quat.y, quat.z]
+    obj.select_set(False)
+    return xyz, quat
+
+
+def get_relative_pose_and_center(obj, scale_factor=np.ones(3)):
```

Comment:
- Refactor into a read-only function (already implemented above) and a write-only function called `reset_pose(obj)` or something. The read-and-write responsibilities of this function are confusing.

Reply:
Actually, are these helper functions even used anywhere? If not, we should remove.